panel/pane/base.py

```diff
@@ -441,14 +450,14 @@ def _cleanup(self, root=None):
         self._inner_layout._cleanup(root)
         super()._cleanup(root)
 
-    def select(self, selector=None):
+    def select(self, selector: Union[type, Callable, None]=None) -> List[Viewable]:
```

Comment:
As far as I can infer it might also be `List[Panel, ReplacementPane]]` but since the docstrings stated `List[Viewable]`, I used that.